content/en/blog/2025/infernece-extension-support/index.md

```diff
@@ -0,0 +1,155 @@
+
+{{< image width="100%"
+    link="./inference-request-flow.svg"
+    alt="Flow of an inference request with gateway-api-inference-extension routing."
+    >}}
+
+## **The Road Ahead**
+
+**Support for Waypoints** - As Istio continues to evolve toward ambient mesh architecture, inference-aware routing will be integrated into waypoint proxies to provide centralized, scalable policy enforcement for AI workloads.
+
+Beyond Istio-specific innovations, the Gateway API Inference Extension community is also actively developing several advanced capabilities that will further enhance routing for for AI inference workloads on Kubernetes:
+
+**HPA Integration for AI Metrics**: Horizontal Pod Autoscaling based on model-specific metrics rather than just CPU and memory.
+
+**Multi-Modal Input Support**: Optimized routing for large multi-modal inputs and outputs (images, audio, video) with intelligent buffering and streaming capabilities.
+
+**Heterogeneous Accelerator Support**: Intelligent routing across different accelerator types (GPUs, TPUs, specialized AI chips) with latency and cost-aware load balancing.
+
+## **Getting Started with Istio Inference Extension**
+
+Ready to try inference-aware routing with Istio? The implementation is officially available starting release 1.27!
+
+For installation and guides, please follow the Istio specific guidance in the [Gateway API Inference Extension website](https://gateway-api-inference-extension.sigs.k8s.io/guides/#__tabbed_3_2).
+
+## **Performance Impact and Benefits**
+
+Early evaluations show significant performance improvements with inference-aware routing, including substantially lower p90 latency at higher query rates and reduced end-to-end tail latencies compared to traditional load balancing.
+
+For detailed benchmark results and methodology, see the [Gateway API Inference Extension performance evaluation](https://kubernetes.io/blog/2025/06/05/introducing-gateway-api-inference-extension/#benchmarks) with testing data using H100 GPUs and vLLM deployments.
+
+The integration with Istio's existing infrastructure means these benefits come with minimal operational overhead, and your existing monitoring, security, and traffic management configurations continue to work unchanged.
+
+## **Conclusion**
+
+The Gateway API Inference Extension represents a significant step forward in making Kubernetes truly AI-ready, and Istio's implementation brings this intelligence to the service mesh layer where it can have maximum impact. By combining inference-aware routing with Istio's proven security, observability, and traffic management capabilities, we're enabling organizations to run AI workloads with the same operational excellence they expect from their traditional services.
+
+---
+
+*Have a question or want to get involved? Join the discussion on Kubernetes Slack - [#gateway-api-inference-extension](https://kubernetes.slack.com/archives/C08E3RZMT2P) channel or on [the Istio Slack](https://slack.istio.io).*
```

Comment:
That first link will only work if you're already on the Kubernetes Slack.  Perhaps something like
```suggestion
*Have a question or want to get involved? [Join the Kubernetes Slack](http://slack.kubernetes.io/ and then find us on the [#gateway-api-inference-extension](https://kubernetes.slack.com/archives/C08E3RZMT2P) channel or [discuss on the Istio Slack](https://slack.istio.io).*
```

Reply:
good callout!